.storybook/preview.js

```diff
@@ -19,7 +19,18 @@ addParameters({
 });
 
 addDecorator(function PaddingDecorator(story) {
-  return <div style={{padding: '8px'}}>{story()}</div>;
+  const storyProps = story().props;
+  const containsFrame =
+    storyProps &&
+    (storyProps.hasOwnProperty('topBar') ||
+      storyProps.hasOwnProperty('navigation') ||
+      storyProps.hasOwnProperty('data-has-frame'));
```

Comment:
This is sort of a hacky way to detect if a frame is present and then removing the padding below. This seems to only work in playgrounds such as DetailsPage.tsx and not the frame stories. Open to alternatives!

Reply:
I think this works just fine